payments-core/src/main/java/com/stripe/android/googlepaylauncher/GooglePayLauncherActivity.kt

```diff
@@ -165,6 +157,26 @@ internal class GooglePayLauncherActivity : AppCompatActivity() {
         }
     }
 
+    private fun onError(
+        taskResult: ApiTaskResult<PaymentData>,
+        errorEvent: ErrorReporter.ErrorEvent,
+        failedResult: GooglePayLauncher.Result.Failed
+    ) {
+        val status = taskResult.status
+        val statusMessage = status.statusMessage.orEmpty()
+        val statusCode = status.statusCode.toString()
+
+        errorReporter.report(
+            errorEvent,
+            additionalNonPiiParams = mapOf(
+                "status_message" to statusMessage,
+                "status_code" to statusCode,
```

Comment:
I don't think these two parameters can be queried from our tables in Hubble but we will see them in Splunk. Should we add these parameters or move them to a formatted `error_message`?

Reply:
Oh that's too bad. What would we be adding them to? Does the error message show up in hubble?

Reply:
All parameters not explicitly captured by a column should end up in the `event_data` column.